app/controllers/documents/send_requested_documents_later_controller.rb

```diff
@@ -5,10 +5,17 @@ class SendRequestedDocumentsLaterController < DocumentUploadQuestionController
 
     def edit
       documents_request = DocumentsRequest.find_by(id: session[:documents_request_id])
-      intake = documents_request.intake
-      documents_request.documents.update_all(intake_id: intake.id)
-      SendRequestedDocumentsToZendeskJob.perform_later(intake.id)
-      redirect_to documents_requested_documents_success_path
+      if documents_request.nil?
+        redirect_to(
+          root_path,
+          alert: t("controllers.send_requested_documents_later_controller.not_found")
```

Comment:
I believe this will style it as a yellow or red banner, as if it were an error or warning. However, the content sounds more like a success confirmation. Should we be using alert or notice?